api/Document.json

```diff
@@ -9955,7 +9955,6 @@
       },
       "wasDiscarded": {
         "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/width",
```

Comment:
Looks like the URL was simply removed on this, rather than adjusted.  I looked and found there's no page for this one, which is something we should probably look into some more later on.  For now, we should probably keep an MDN URL around?

Reply:
> For now, we should probably keep an MDN URL around?

What value should it have? Pointing `wasDiscarded` to `width` seems pointless; should it be `Document`?

Reply:
I'd say the value should be `https://developer.mozilla.org/en-US/docs/Web/API/Document/wasDiscarded` -- even if the page doesn't exist right now.  (We've got a LOT of pages to create, apparently...)

Reply:
For now, I made it `https://developer.mozilla.org/en-US/docs/Web/API/Document` so that people don't experience as many broken URLs.